docs/2.10.0/docs/canton/usermanual/FAQ.rst

```diff
@@ -131,8 +131,10 @@ Why is Canton complaining about my database version?
 Postgres
 ~~~~~~~~
 
-Canton is tested with Postgres 10, 11, 12, 13, and 14 -- so these are the recommended versions. Canton is also likely to work with any higher versions,
-but will WARN when a higher version is encountered. By default, Canton will not start when the Postgres version is below 10.
+Canton is tested with the following PostgreSQL releases:  12, 13, 14, and 15.
+Newer PostgreSQL releases may not have been tested. The recommended version
```

Comment:
Don't we want something stronger since we know that it will not start with postgres 16 ?

Reply:
A fair question.  My experience is that customers will stay away from anything that hasn't been tested.

For those brave customers that do and Canton won't start, the support team can point them to this.  So we should be covered.

I'm a little reluctant to make to hard a statement because if we do enable postgres 16, then someone needs to remember to update the docs.  I'm not sure we have that type of process in place so making the statement a little weak will allow for the situation where:  (1) we allow for postgres 16; (2) there is a brave customer who tries it and it works.